.github/workflows/images.yml

```
@@ -76,14 +83,14 @@ jobs:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           flavor: |
             latest=false
-            suffix=-linux-python${{ matrix.python }}
+            suffix=-linux-python${{ matrix.python }}${{ matrix.channel == 'conda-forge' && '-conda-forge' || '' }}
```

I don't think we've agreed to maintain the main image name that way, I could just see this being 
```suggestion
            suffix=-linux-python${{ matrix.python }}-${{ matrix.channel }}
```

---

Hmpf, yeah, true. Okay, fair enough.